pkg/limayaml/defaults.go

```diff
@@ -51,54 +51,136 @@ func MACAddress(uniqueID string) string {
 	return hw.String()
 }
 
-func FillDefault(y *LimaYAML, filePath string) {
-	y.Arch = resolveArch(y.Arch)
+// FillDefault updates undefined fields in y with defaults from d (or built-in default), and overwrites with values from o.
+// Both d and o may be empty.
```

Comment:
We should clarify how `d` and `o` are prepended/appended to slice fields of `y` (e.g., `mounts`, `containerd.archives`)